src/log.cpp

```diff
@@ -35,43 +35,58 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include <cerrno>
 #include <cstring>
 
-const int BUFFER_LENGTH = 256;
-
-class StringBuffer : public std::streambuf {
+class ChangableTarget : public LogTarget {
```

Comment:
This appears to be unused, neither `null_stream` not `null_target` are used.

Reply:
True... I will remove ChangableTarget and the null streams.

I had originally used ChangableTarget for dout_con / derr_con, because they were being modified like so:

https://github.com/minetest/minetest/blob/master/src/main.cpp#L460

But after adding the trace logging level, this is no longer necessary.